Sources/Storyboard/StoryboardSceneBased.swift

```diff
@@ -49,9 +49,13 @@ public extension StoryboardSceneBased where Self: UIViewController {
    - returns: instance of the conforming ViewController
    */
   static func instantiate() -> Self {
-    guard let vc = storyboard.instantiateViewControllerWithIdentifier(self.sceneIdentifier) as? Self else {
-      fatalError("The viewController '\(self.sceneIdentifier)' of '\(storyboard)' is not of class '\(self)'")
+    let sb = Self().storyboard
+
+    let vc = sb?.instantiateViewController(withIdentifier: self.sceneIdentifier)
```

Comment:
Most of let in other methods have well descriptive names. Here are `sb`, `vc`, `typedVC`. IMO you could use full names ;)